styx-common/src/main/java/com/spotify/styx/storage/DatastoreStorage.java

```diff
@@ -377,14 +398,18 @@ public void updateNextNaturalTrigger(WorkflowId workflowId, TriggerInstantSpec t
   }
 
   /**
-   * Eventually consistently list all active states and strongly consistently fetch their values.
-   *
-   * <p>This method will return a map of active states that might be missing some recently created
-   * states, but the values of all the states returned should be fresh.
+   * Strongly consistently read all active states
```

Comment:
Maybe nitpick, but "strongly consistently" might be misleading. I too lack a short term for the actual consistency mode :/ Thought to go for something like this: https://github.com/spotify/styx/compare/uldis/strong-listing#diff-1ec84406994237ab0330232416a42f1eR477

Reply:
"strongly consistently" matches the datastore documentation terminology and (I believe) the terminology in the distributed system community as well

Reply:
Yes if you're talking about each individual lookup. My point is that this situation is not a lookup but a series of lookups used to obtain one list. Calling the list "strongly consistent" in such a situation is not something that I've seen out there.